src/pages/_theme.vue

```diff
@@ -5,59 +5,159 @@ meta:
+        </div>
+        <div class="m-2">
+          <AppButton variant="linkOutlined">linkOutlined</AppButton>
+        </div>
+        <div class="m-2">
+          <AppButton variant="dangerOutlined">dangerOutlined</AppButton>
+        </div>
+        <div class="m-2">
+          <AppButton variant="text">text</AppButton>
+        </div>
+        <div class="m-2">
+          <AppButton variant="dangerText">dangerText</AppButton>
+        </div>
+      </div>
+    </section>
+
+    <section class="my-5">
+      <h3 class="mb-2 font-title text-xl">AppSlider</h3>
+      <div class="bg-white p-4">
+        <div class="max-w-[1200px] mx-auto relative">
+          <AppSlider :show-navigation-buttons="true" :infinite="true">
+            <!-- Put your slides here -->
+            <template #slides>
+              <div class="flex-none w-full bg-danger">
+                <img
+                  class="mx-auto max-w-full"
+                  src="https://picsum.photos/seed/1/1200/300"
+                  alt="Lorem Picsum"
+                />
+              </div>
+              <div class="flex-none w-full bg-success">
+                <div class="w-full h-full flex justify-center items-center">
+                  <p>You can put anything you want in here!</p>
+                </div>
+              </div>
+              <div class="flex-none w-full bg-warning">
+                <img
+                  class="mx-auto max-w-full"
+                  src="https://picsum.photos/seed/2/1200/300"
+                  alt="Lorem Picsum"
+                />
+              </div>
+            </template>
+
+            <!-- Define your custom navigation template here -->
+            <template #navigation="{ nextSlide, prevSlide, toSlide }">
+              <span class="w-full h-full flex justify-between absolute top-0">
+                <button class="" @click="prevSlide()">
+                  <font-awesome-icon
+                    class="text-2xl p-2 text-white hover:text-link"
+                    :icon="PrevIcon"
+                  />
+                </button>
+                <button @click="nextSlide()">
+                  <font-awesome-icon
+                    class="text-2xl p-2 text-white hover:text-link"
+                    :icon="NextIcon"
+                  />
+                </button>
+              </span>
+
+              <button
+                class="text-2xl p-2 text-white hover:text-link absolute top-0 left-0"
+                @click="toSlide(0)"
+              >
+                <font-awesome-icon :icon="ResetIcon" />
+              </button>
+            </template>
+          </AppSlider>
+        </div>
+      </div>
+    </section>
+  </section>
 </template>
 
 <script lang="ts" setup>
 import AppColor from '../components/AppColor.vue';
+import AppButton from '../components/button/AppButton.vue';
+import AppSlider from '../components/AppSlider.vue';
+import {
+  faAngleRight as NextIcon,
```

Comment:
Not a big thing but I think it's a bit clearer and more consistent with the rest of the codebase to reference these directly rather than creating an alias

Reply:
I did it this way because I could easily test other icons and only had to change the code in one place, but I can adapt it if you like.

Reply:
I like the idea but I think for now let's just do simple imports for now. It would be good to have another file that maps Font Awesome icons to our "vocabulary", I'll add a task to the backlog.